mappings/net/minecraft/util/annotation/ClientMethodsReturnNonnullByDefault.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_6179 net/minecraft/util/annotation/ClientMethodsReturnNonnullByDefault
+	COMMENT Specifies that all methods in the annotated package/class return nonnull
+	COMMENT values unless nullability is specified with a separate annotation.
+	COMMENT
+	COMMENT <p>While this annotation is meant to be a package/class annotation, it can
```

Comment:
Why do we need this disclaimer? If it's not restructed, what evidence do you have to assert what it was "meant to be"?

Reply:
Because as a `@TypeQualifierDefault` it has no effect on anything that's not a package or class.